lib/index.js

```diff
@@ -36,37 +36,38 @@ export default function runContentfulExport (usageParams) {
 
         log.info('Downloading ' + response.assets.length + ' assets')
 
-        var arrayOfDownloadPromises = []
-        response.assets.forEach((asset) => {
+        Promise.map(response.assets, (asset) => {
```

Comment:
The promised returned from `Promise.map` is not awaited which means that any following code (https://github.com/contentful/contentful-export/blob/8d8cc10710f089213d4db0a2f8a599f4d880c1a2/lib/index.js#L74) will be executed immediately without waiting for the donwload of the assets. Shouldn't we wait for it to finish?